src/libraries/System.Private.CoreLib/src/System/Globalization/Normalization.Unix.cs

```diff
@@ -20,7 +22,14 @@ internal static bool IsNormalized(string strInput, NormalizationForm normalizati
 
             ValidateArguments(strInput, normalizationForm);
 
-            int ret = Interop.Globalization.IsNormalized(normalizationForm, strInput, strInput.Length);
+            int ret;
+            unsafe
```

Comment:
>unsafe [](start = 12, length = 6)

nit: would be more readable if you mark the method unsafe and avoid introducing unsafe block here. this apply to the other method too.

Reply:
That is also my preference, I was under the assumption that the style in runtime is that methods shouldn't be marked unsafe if their inputs don't technically require it. Changed.